internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -866,6 +867,128 @@ func TestAccClusterAdvancedCluster_priorityNewSchema(t *testing.T) {
 	})
 }
 
+func TestAccAdvancedCluster_oldToNewSchemaWithAutoscalingDisabled(t *testing.T) {
+	acc.SkipIfTPFAdvancedCluster(t)
```

Comment:
it's been renamed in master to acc.SkipIfAdvancedClusterV2Schema

Reply:
We'll keep it in mind when we merge master to the dev branch 👍